core/controllers/email_dashboard.py

```diff
@@ -143,6 +173,40 @@ def get(self):
 class EmailDashboardResultPage(base.BaseHandler):
     """Handler for email dashboard result page."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'query_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'email_subject': {
```

Comment:
email_subject, email_body, email_intent & max_recipients was part of "data", I have updated frontend to pass these params individually instead of type dict, because -- max_recepients is an optional param and we do not have a way to introduce a optional param in type dict.
Also, currently a number of issues are present in frontend files which I have updated, ptal.

Reply:
I have added a video regarding the error.